src/test/kotlin/uk/gov/dluhc/printapi/rest/GetAnonymousElectorDocumentSummaryByApplicationIdIntegrationTest.kt

```diff
@@ -0,0 +1,187 @@
+                aedDocumentWithDifferentGssCode,
+                aedDocumentWithDifferentSourceType
+            )
+        )
+
+        val expectedRecord1 = with(aedMatchingDocument1) {
+            buildAnonymousElectorDocumentSummary(
+                certificateNumber = certificateNumber,
+                electoralRollNumber = electoralRollNumber,
+                gssCode = gssCode,
+                certificateLanguage = EN,
+                supportingInformationFormat = STANDARD,
+                deliveryAddressType = DeliveryAddressType.REGISTERED,
+                elector = with(contactDetails!!) {
+                    buildAnonymousElector(
+                        addressee = "John Jacob",
+                        registeredAddress = with(address!!) {
+                            buildValidAddress(
+                                property = property,
+                                street = street!!,
+                                town = town,
+                                area = area,
+                                locality = locality,
+                                uprn = uprn,
+                                postcode = postcode!!
+                            )
+                        }
+                    )
+                },
+                status = PRINTED,
+                photoLocation = photoLocationArn,
+                issueDate = issueDate,
+                userId = userId,
+                dateTime = requestDateTime.atOffset(ZoneOffset.UTC),
+            )
+        }
+        val expectedRecord2 = with(aedMatchingDocument2) {
+            buildAnonymousElectorDocumentSummary(
+                certificateNumber = certificateNumber,
+                electoralRollNumber = electoralRollNumber,
+                gssCode = gssCode,
+                certificateLanguage = EN,
+                supportingInformationFormat = STANDARD,
+                deliveryAddressType = DeliveryAddressType.ERO_MINUS_COLLECTION,
+                elector = with(contactDetails!!) {
+                    buildAnonymousElector(
+                        addressee = "Mike William Brown Johnson",
+                        registeredAddress = with(address!!) {
+                            buildValidAddress(
+                                property = property,
+                                street = street!!,
+                                town = town,
+                                area = area,
+                                locality = locality,
+                                uprn = uprn,
+                                postcode = postcode!!
+                            )
+                        }
+                    )
+                },
+                status = PRINTED,
+                photoLocation = photoLocationArn,
+                issueDate = issueDate,
+                userId = userId,
+                dateTime = requestDateTime.atOffset(ZoneOffset.UTC),
+            )
+        }
```

Comment:
Minor - this is a good test, but the given section is quite long. Might be worth seeing if it can be condensed a little, but I appreciate you don't want to sacrifice the quality of the test.